Modules/Core/Common/include/itkSymmetricEigenAnalysis.h

```diff
@@ -865,11 +862,11 @@ class ITK_TEMPLATE_EXPORT SymmetricEigenAnalysisFixedDimension
   {
   using ValueType = decltype(GetMatrixValueType(true));
   using EigenLibMatrixType =
-    Eigen::Matrix< ValueType, m_Dimension, m_Dimension, Eigen::RowMajor>;
-  EigenLibMatrixType inputMatrix( m_Dimension, m_Dimension);
-  for (unsigned int row = 0; row < m_Dimension; ++row)
+    Eigen::Matrix< ValueType, TMatrixDimension, TMatrixDimension, Eigen::RowMajor>;
+  EigenLibMatrixType inputMatrix( TMatrixDimension, TMatrixDimension);
```

Comment:
This constructor takes 'const unsigned int &'. The matrix is fixed so these parameters are redundant.

Reply:
That constructor was for the Dynamic case, and badly re-used there. The parameters should be removed. Thanks @blowekamp

Reply:
@phcerdan Since this is making our dashboard RED at the moment, I'm committing as is.

@phcerdan Can you please make a separate commit with the improvement?